src/app/client/src/app/modules/workspace/components/collaboration-content-filter/collaboration-content-filter.component.ts

```diff
@@ -132,12 +145,48 @@ export class CollaborationContentFilterComponent implements OnInit {
     } else {
       delete this.queryParams['query'];
     }
+    this.setCollabartionContentSearchInteractEdata();
     this.route.navigate(['workspace/content/collaborating-on', 1], { queryParams: this.queryParams });
   }
   keyup(event) {
     this.query = event;
     this.modelChanged.next(this.query);
   }
+  setCollabartionContentSearchInteractEdata() {
+    const searchInteractEdata = {
+      id: 'search-collabaration-content',
+      type: 'click',
+      pageid: 'collabratingon'
+    };
+    if (this.query) {
+      searchInteractEdata['extra'] = {
+        query: this.query
+      };
+    }
+    this.telemetryInteractEdata = searchInteractEdata;
+    if (this.telemetryInteractEdata) {
+      this.appTelemetryInteractData = {
+       context: {
+          env: _.get(this.telemetryInteractContext, 'env') || _.get(this.activatedRoute, 'snapshot.root.firstChild.data.telemetry.env') ||
```

Comment:
env : _.get(this.activatedRoute, 'snapshot.data.telemetry.env') no need for other cases